hc/accounts/models.py

```diff
@@ -56,17 +57,23 @@ def set_api_key(self):
     def send_report(self):
         # reset next report date first:
         now = timezone.now()
-        self.next_report_date = now + timedelta(days=30)
+        if self.report_period == 0:  # monthly report
```

Comment:
Can you use enum here for the purpose of understanding the types of report period

Reply:
We are fixing this